lib/gitlab/client.rb

```diff
@@ -15,14 +15,18 @@ class Client < API
     include Builds
     include Commits
     include Deployments
+    include DockerfileTemplates
     include Environments
     include Events
+    include GitignoreTemplates
+    include GitlabCiYmlTemplates
```

Comment:
Can we merge all these modules into one `Templates` module? Because technically it's one endpoint: `/templates`. Except ProjectTemplates module - it's methods should go to Projects module.

Reply:
Made changes. Need review.